### DISCUSSION TITLE
I need 1 MORE!! PLEASE Tell me your favorite Movie!!

### DISCUSSION THREAD

**Discussion by Daisuke134** [NAIST-SE/AttractingContributors#1025] [2024 Attraction Contributions]
https://github.com/Daisuke134/AttractingContributors-Movies

Hi. My name is Daisuke, and love movies very much!!
Please share with me your favorite movies so I can discover and enjoy them too.

**Comment by 37integer** [comment]
Interstellar

**Reply by Daisuke134** [reply]
@37integer Thank you, but please add it to my Readme page. Do you know how to do it? Also, I already have interstellar on the list. Would be great if you could add something else not on the list!!

**Reply by Daisuke134** [reply]
@37integer Just go to my link and edit the Readme file→add the movie name→Commit changes→Pull Request!!

**Reply by 37integer** [reply]
ok, I add other movie

**Reply by 37integer** [reply]
I already do pull request, can you see?

**Reply by Daisuke134** [reply]
Yes I can! I merged it! Thank you. Inception is also a great movie!!
